frontend/online-shop/src/App.tsx

```diff
@@ -1,26 +1,30 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { TableCards } from "./components/productsList/TableCards";
+import { ProductDetails } from "./components/productDetails/ProductDetails";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from "react-router-dom";
+import { ThemeProvider } from "@material-ui/styles";
+import { standardTheme } from "./themes/StandardTheme";
+import { CssBaseline } from "@material-ui/core";
 
-function App() {
+export const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <ThemeProvider theme={standardTheme}>
+      <CssBaseline>
+        <Router>
+          <Switch>
+            <Route path="/product/:number" component={ProductDetails}></Route>
+            <Route exact path="/">
+              <Redirect to="/products" />
+            </Route>
+            <Route path="/products" component={TableCards}></Route>
```

Comment:
Try to be a little more concise with the naming of components. `TableCards` is very confusing especially because it does not have the word `Products` in it. Maybe `ProductList` or `ProductCards` would be a better fit.